tests/test_common.py

```diff
@@ -20,6 +20,9 @@ def test_factorial_float_raise_error(self):
         with self.assertRaises(the_exception.InputError):
             common.factorial(56.3)
 
+    def test_factorial_float_raise_error(self):  # 0b is removed from bin result for all unit tests
+        self.assertEqual((common.convert_to_binary(6)), bin(6).replace("0b", ""))  # Test for positive integer
+        self.assertEqual((common.convert_to_binary(-6)), bin(-6).replace("0b", ""))  # Test for negative integer
```

Comment:
How come we aren't testing floating point values?

Reply:
This is something that I need to add but was going to do it in the next PR as I have to figure out how because the bin() function doesn't accept floating point values.

Reply:
Ah ok